views/backend/postback.php

```diff
@@ -26,24 +25,27 @@
         }
 
         if(isset($_POST['submit'])){
-            $title = htmlspecialchars(trim($_POST['title']));
-            $content = htmlspecialchars(trim($_POST['content']));
+            if(isset($_POST['title'])){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Direct use of $_POST Superglobal detected.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5344015)